src/styles/main.scss

```diff
@@ -118,8 +118,8 @@ $primary-disabled: lighten( map-get($theme-colors,"secondary"), 20% );
 }
 
 .prop-heading {
-  background-color: #8C8A83 !important;
-  color: white !important;
+  background-color: #D0C1A8 !important;
```

Comment:
Can we stick this color in a variable at the top of the file?

Reply:
Done.